### PR TITLE
added kubernetes demo for conf_ml

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: samples/confidential_ml/confml_server.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        volumeMounts:
+        - name: var-run-aesmd
+          mountPath: /var/run/aesmd # hardcoded in intels libarary, volume name must match that of the daemonset 
+        ports:
+        - name: client1
+          containerPort: 8000
+        imagePullPolicy: Always
+        resources:
+          limits:
+            kubernetes.azure.com/sgx_epc_mem_in_MiB: "100"
```
Comment (vtikoo) [inline samples/confidential_ml/confml_server.yaml:52]: It might be better to have ICX defaults rather than CFL.

**Review by vtikoo — APPROVED** [review]
Thanks for extending this sample to AKS, great work!

File: samples/confidential_ml/confml_server.yaml (hunk 1 above)
Comment (salsal97) [reply samples/confidential_ml/confml_server.yaml:52]: In the demo we gave them both 100MiB

Comment (salsal97) [reply samples/confidential_ml/confml_server.yaml]: Bumped it to 4GiB to match the config.json value as we discussed

**Pull request merged**